functions/scheduleinstance/index.js

```diff
@@ -15,7 +15,18 @@
 // [START functions_start_instance_pubsub]
 // [START functions_stop_instance_pubsub]
 const compute = require('@google-cloud/compute');
-const instancesClient = new compute.InstancesClient({fallback: 'rest'});
+const instancesClient = new compute.InstancesClient();
+const operationsClient = new compute.ZoneOperationsClient();
+
+async function waitForOperation(projectId, operation) {
```

Comment:
@alexander-fenster is this really the best way to wait for an operation to complete?  I thought we integrated Compute's weird LRO into our existing framework?

Reply:
@summer-ji-eng is working on it; we only have automatic polling for a few languages, others (including TypeScript) are coming soon, in a non-breaking way.

Reply:
Summer, from the future compatibility point of view, would this be the best way to wait for the operation?